ethers-providers/Cargo.toml

```diff
@@ -24,6 +24,7 @@ serde_json = { version = "1.0.64", default-features = false }
 thiserror = { version = "1.0.24", default-features = false }
 url = { version = "2.2.1", default-features = false }
 auto_impl = { version = "0.4.1", default-features = false }
+ahash = { version = "0.7", default-features = false, features = ["std"], optional = true }
```

Comment:
If you really need higher performance than `SipHasher` (I would suggest measuring), and don't care about hashDoS (the `ahash` crate is not designed to be hashDoS resistant), I would suggest using the [`twox-hash` crate](https://crates.io/crates/twox-hash) which implements the XXHash algorithm.

Reply:
Yeah I'm aware of this, but figured this usage wouldn't require that property, but maybe I'm overlooking some concern. Thanks for the link, I'll look into that, but I'll just change to whatever @gakonst prefers since I don't have an opinion.

Reply:
I've switched it to siphash as a default, but happy to switch to whatever is preferred :)

Reply:
Sounds good. I think using what the standard library provides is a reasonable starting point, and if you need better performance, it would be good to measure whether a 3rd-party hashing crate actually provides an improvement.